CONTRIBUTING.md

```diff
@@ -24,71 +24,38 @@ In order to build **[nimi-python](https://github.com/ni/nimi-python)**, you must
 following installed:
 
 * [Python](https://www.python.org/downloads/)
-    - If you're on Windows (Not Windows Subsystem for Linux)
-        - Install Python 2.7 x64
-        - Install at least one Python 3.x x64 - 3.4, 3.5 or 3.6 will all work
-        - Optional - 32 bit versions
-        - Ensure pip support is installed for all versions
-        - Install paths can either be in the appropriate Program Files for the bitness, or c:\pythonXY for 64 bit and c:\pythonXY-32 for 32 bit
+    - If you're on Windows 10 (not [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about))
+        - Install at least one 64-bit version of Python. The current supported versions are 3.6, 3.7 and 3.8
```

Comment:
Change to:
```
Install the latest 64-bit Python version supported by nimi-python. Refer to the README for supported versions.
```

..since most of the steps in tox.py require the latest supported Python version.